app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -3498,6 +3498,221 @@ def specific_arg_info
+      end
+
+      def arg_defaults
+        {
+          :note => 60,
+          :note_slide => 0,
+          :note_slide_shape => 1,
+          :note_slide_curve => 0,
+          :amp => 1,
+          :amp_slide => 0,
+          :amp_slide_shape => 1,
+          :amp_slide_curve => 0,
+          :pan => 0,
+          :pan_slide => 0,
+          :pan_slide_shape => 1,
+          :pan_slide_curve => 0,
+
+          :attack => 0.01,
+          :decay => 0,
+          :sustain => 1,
+          :release => 0.01,
+          :attack_level => 1,
+          :decay_level => :sustain_level,
+          :sustain_level => 1,
+
+          :bass => 8,
+          :bass_slide => 0,
+          :bass_slide_shape => 1,
+          :bass_slide_curve => 0,
+
+          :quint => 8,
+          :quint_slide => 0,
+          :quint_slide_shape => 1,
+          :quint_slide_curve => 0,
+
+          :fundamental => 8,
+          :fundamental_slide => 0,
+          :fundamental_slide_shape => 1,
+          :fundamental_slide_curve => 0,
+
+          :oct => 8,
+          :oct_slide => 0,
+          :oct_slide_shape => 1,
+          :oct_slide_curve => 0,
+
+          :nazard => 0,
+          :nazard_slide => 0,
+          :nazard_slide_shape => 1,
+          :nazard_slide_curve => 0,
+
+          :blockflute => 0,
+          :blockflute_slide => 0,
+          :blockflute_slide_shape => 1,
+          :blockflute_slide_curve => 0,
+
+          :tierce => 0,
+          :tierce_slide => 0,
+          :tierce_slide_shape => 1,
+          :tierce_slide_curve => 0,
+
+          :larigot => 0,
+          :larigot_slide => 0,
+          :larigot_slide_shape => 1,
+          :larigot_slide_curve => 0,
+
+          :sifflute => 0,
+          :sifflute_slide => 0,
+          :sifflute_slide_shape => 1,
+          :sifflute_slide_curve => 0,
+
+          :rs_freq => 6.7,
+          :rs_freq_slide => 0,
+          :rs_freq_slide_shape => 1,
+          :rs_freq_slide_curve => 0,
+
+          :rs_freq_var => 0.1,
+          :rs_pitch_depth => 0.008,
+          :rs_delay => 0,
+          :rs_onset => 0,
+          :rs_pan_depth => 0.05,
+          :rs_amplitude_depth => 0.2,
+        }
+      end
+
+      def specific_arg_info
+        {
+          :note =>
+          {
+            :doc => "Note to play. Either a MIDI number or a symbol representing a note. For example: `30`, `52`, 56.5, `:C`, `:C2`, `:Eb4`, or `:Ds3`. This synth does allow changing or sliding the note while playing. In real tonewheel organs one would get this effect only when fiddling with the motor driving the tonewheel.",
+            :validations => [v_positive(:note), v_less_than(:note, 231)],
+            :modulatable => true
+          },
+          :bass =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component one octave below the base tone, i. e. half its frequency",
```

Comment:
`i.e.` is written without a space between the letters... (same for the other instances here)

Reply:
Fixed.